README.md

```diff
@@ -6,8 +6,8 @@
 
 <div align="center">
   
-  [![license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/iTwin/iTwinUI-react/blob/main/LICENSE.md)
```

Comment:
Removed because it's redundant

GitHub:
![image](https://user-images.githubusercontent.com/9084735/115306578-08a0e480-a136-11eb-9e4b-d301758a9225.png)

Npm:
![image](https://user-images.githubusercontent.com/9084735/115306631-1f473b80-a136-11eb-8ca9-944dec27e153.png)